app/components/search/SearchMap/SearchMap.tsx

```diff
@@ -2,17 +2,20 @@ import React, { ReactElement } from 'react';
 import GoogleMap from 'google-map-react';
 import { Tooltip } from 'react-tippy';
 import 'react-tippy/dist/tippy.css';
-import SearchEntry from './SearchEntry';
-import config from '../../../config';
+import SearchEntry from 'components/search/SearchMap/SearchEntry';
+import { useAppContext } from 'utils';
+import { createMapOptions, UserLocationMarker, CustomMarker } from 'components/ui/MapElements';
 import './SearchMap.scss';
-import { useAppContext } from '../../../utils';
-import { createMapOptions, UserLocationMarker, CustomMarker } from '../../ui/MapElements';
 import { SearchHit } from '../../../models';
+import config from '../../../config';
 
-export const SearchMap = ({ hits, hitsPerPage, page }: {
-  hits: SearchHit[];
-  hitsPerPage: number;
-  page: number;
+export const SearchMap = ({
+  hits, hitsPerPage, page, setMapObject,
+}: {
+    hits: SearchHit[];
+    hitsPerPage: number;
+    page: number;
+    setMapObject: Function;
```

Comment:
Could we change this to a more specific function signature, even if we don't know exactly what a map object is? This at least makes it clearer that there's a single argument and that the return type is void.

```suggestion
    setMapObject: (map: any) => void;
```

Reply:
Yep, missed that one. thanks!